examples/build.gradle

```diff
@@ -38,6 +46,29 @@ allprojects {
         jcenter()
         google()
     }
+
+    if (!project.name.startsWith("realm-examples")
+            && !project.name.startsWith("library")
+            && !project.name.startsWith("moduleExample")) { // exclude root and library project
+        task runMonkey(dependsOn: 'installDebug') {
```

Comment:
we still need to run the `monkeyRelease` after uploading to bintrary. This doesn't seem to handle it?

Reply:
I don't quite understand this, the task to run monkey for example is still `:monkeyDebug` and `:monkeyRelease`?

Reply:
updated the script, now it has the same name as the previous tasks, also generating a release & debug tasks